test/uk/gov/hmrc/transitmovementsconverter/services/ConverterServiceSpec.scala

```diff
@@ -28,6 +28,7 @@ import play.api.libs.json.JsObject
 import play.api.libs.json.Json
```

Comment:
Needs rename to ConversionServiceSpec

Reply:
Actually, turns out that we have the `ConverterService`, so this is fine, but the integration test was not!